cloudevents/extensions/jwt.md

```diff
@@ -0,0 +1,147 @@
+# JWT Extension
+
+This extension provides a way to map JWT (JSON Web Token) values in the formats of JWE
+ (JSON Web Encryption) or JWS (JSON Web Signature).
+ 
+JWE MAY be used to encrypt a CloudEvent data and JWS MAY be used to sign CloudEvent data
+
+This extension does not support [JWE Json Serialization][jwe-json-serialization] 
+
+When using [JWE Cipher Text][jwe-ciphertext] MUST be mapped onto `data` and
+ `datacontenttype` MUST be set to `application/octet-stream`
+
+When using JWS the signed data MUST be decoded from base64url format and mapped onto
+`data` in `Binary` format where `datacontenttype` is taken from the `cty` property
+of the JOSE header, if no `cty` property exists `datacontenttype` MUST be set to `application/octet-stream`
+  
+## Attributes
+
+### jose
+- Type: `Binary`
+- Description: utf-8 encoded JOSE Header value.
+    Both JWS and JWE have JOSE Header definitions.
+- Constraints:
+  - REQUIRED
+  - MUST be a non-empty octet sequence
```

Comment:
It may feel redundant since we say this is Binary, it might be more clear if we say:
```
- MUST be a base64-encoded non-empty octet sequence
```
We do this for `time` in the main spec too just to be extra clear that it must adhere to the time RFC

Reply:
Same for the other Binary properties below

Reply:
No, it is confusing, because you will think you will need to encode it to base64 and then put the result as the binary content